log_level.go

```diff
@@ -0,0 +1,31 @@
+package watchman
+
+// the supported log levels
+const (
+	DebugLogLevel = "debug"
+	ErrorLogLevel = "error"
+	OffLogLevel   = "off"
+)
```

Comment:
can you rename these to `LogLevelDebug`, `LogLevelError`, etc...

Reply:
@jonasi good call, updated. Do you have any thoughts on using making these have a distinct type `type LogLevel string`? That's how I had it originally but I ran in to the issue where we weren't correctly handling type aliases.